app/models/space.rb

```diff
@@ -59,6 +62,23 @@ def someone_else_editing?(current_user, previous_updated_at_str)
     checkpoints.last.author_id != current_user.id
   end
 
+  def guesstimate_expressions
+    return [] unless graph.present? && graph["guesstimates"].present?
+    graph["guesstimates"].map { |g| g["expression"] }
+  end
+
+  def get_imported_fact_ids()
+    guesstimate_expressions.map {|e| e.scan(/\$\{fact:(\d+)/) unless e.blank? }.flatten.uniq.keep_if { |e| e.present? }
+  end
+
+  def update_exported_facts_count!()
+    update_columns(exported_facts_count: exported_facts.count)
+  end
+
+  def update_imported_facts!()
```

Comment:
Call this `update_imported_fact_ids` and change the parameter name to `imported_fact_ids`